dfp/create_creatives.py

```diff
@@ -49,17 +51,21 @@ def create_creative_config(name, advertiser_id):
 
   # https://developers.google.com/doubleclick-publishers/docs/reference/v201802/CreativeService.Creative
   config = {
-    'xsi_type': 'ThirdPartyCreative',
     'name': name,
     'advertiserId': advertiser_id,
-    'size': {
-      'width': '1',
-      'height': '1'
-    },
-    'snippet': snippet,
-    'isSafeFrameCompatible': True,
   }
 
+  if video_ad_type:
+    config['xsi_type'] = 'VastRedirectCreative'
+    config['duration'] = 1000
+    config['size'] = { 'width': '640', 'height': '480' }
```

Comment:
Are there additional options that need to be configurable, or is this sufficient for most use cases?

Reply:
From my point of view this is sufficient in prebid context, because the size is transmitted with the bid request and the duration is part of the response.